CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 (modification: no type change headlines) and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [4.1.0] - 2020-12-03
```

Comment:
Please always use `UNRELEASED` in occasions of opening a new CHANGELOG entry, this otherwise has some high potential in misleading others to think that `v4.1.0` would have already been released.

(this actually e.g. happened recently on the block library with the `v3` release having a date in CHANGELOG but wasn't released for months which at some point was the source for a high level of confusion)

Reply:
A yes, you are right, it felt a bit strange to also put a date on this one and a version number.